server/src/test/integration/org/apache/cassandra/sidecar/testing/CassandraSidecarTestContext.java

```diff
@@ -184,21 +184,21 @@ public void setSslConfiguration(SslConfiguration sslConfiguration)
         refreshInstancesConfig();
     }
 
-    public InstancesConfig instancesConfig()
+    public InstancesMetadata instancesConfig()
```

Comment:
rename the method here too?
```suggestion
    public InstancesMetadata instancesMetadata()
```